src/vector_of_array.jl

```diff
@@ -510,7 +510,7 @@ function Base.CartesianIndices(VA::AbstractVectorOfArray)
 end
 
 # Tools for creating similar objects
-Base.eltype(::VectorOfArray{T}) where {T} = T
+Base.eltype(::Type{<:AbstractVectorOfArray{T}}) where {T} = T
```

Comment:
don't we want both?

Reply:
`eltype` docs say to define the latter. There already is a dispatch in Base for `eltype(x) = eltype(typeof(x))`

Reply:
👍